Dockerfile

```diff
@@ -1,20 +1,17 @@
 FROM mcr.microsoft.com/dotnet/core/sdk:2.2 AS build
 WORKDIR /app
 
-# copy csproj and restore as distinct layers
-COPY *.csproj .
-RUN dotnet restore
-
-# copy everything else and build app
+# copy and build app
 COPY . ./
-RUN dotnet publish -c Release -o out
+RUN dotnet restore ntbs-service/*.csproj
+RUN dotnet publish ntbs-service/*.csproj -c Release -o out
 
 FROM node AS build-frontend
 WORKDIR /app
 
 # copy package.json and restore as distinct layers
-COPY package.json .
-COPY package-lock.json .
+COPY ntbs-service/package.json .
+COPY ntbs-service/package-lock.json .
 RUN npm install
 
 # copy everything else and build frontend app
```

Comment:
It goes wrong at the npm run stage.. maybe we should actually cd into the ntbs-service rather than referencing it?